language/move-compiler/Cargo.toml

```diff
@@ -2,6 +2,7 @@
 name = "move-compiler"
 version = "0.0.1"
 authors = ["Diem Association <opensource@diem.com>"]
+description = "Compile Move source to Move bytecode"
```

Comment:
I think this actually a bit too specific for the compiler. You could remove it from this PR, or switch it to "The definition of the Move source language, and it's compiler" or something. There is a lot going on in here besides bytecode generation